solr/test-framework/src/java/org/apache/solr/cloud/ZkTestServer.java

```diff
@@ -644,8 +644,10 @@ public void shutdown() throws IOException, InterruptedException {
 
       while (true) {
         try {
-          zooThread.join();
-          ObjectReleaseTracker.release(zooThread);
+          if (zooThread != null) {
```

Comment:
belongs in another issue?

Reply:
This came up while I was trying to debug a failing test, but sure